src/Components/ExtractLevelsMenu.jsx

```diff
@@ -33,6 +33,11 @@ export default function ExtractLevelsMenu({listOfOptions, icon, title}) {
     setAnchorEl(null)
   }
   const viewer = useStore((state) => state.viewerStore)
+  // ADDED
+  const levelInstance = useStore((state) => state.levelInstance)
+  const setLevelInstance = useStore((state) => state.setLevelInstance)
```

Comment:
did you add these variables to the store?
<img width="854" alt="image" src="https://user-images.githubusercontent.com/3433606/194532685-c09d618f-8ddf-4e5c-b9d0-d7134468b3e1.png">
if not add it to the UI slice in the store folder.

Reply:
Addressed. Added to UI Slice